packages/web/src/instrumentations/errors/index.ts

```diff
@@ -1,16 +1,105 @@
-import { BaseInstrumentation, VERSION } from '@grafana/agent-core';
-
-import { registerOnerror } from './registerOnerror';
-import { registerOnunhandledrejection } from './registerOnunhandledrejection';
-
+import { BaseInstrumentation, VERSION, isString, isPrimitive } from '@grafana/agent-core';
 export { parseStacktrace } from './stackFrames';
+import type { ExceptionStackFrame } from '@grafana/agent-core';
+
+import { primitiveUnhandledType, primitiveUnhandledValue , unknownString } from './const';
+import type { ExtendedPromiseRejectionEvent } from './extendedPromiseRejectionEvent';
+import { getErrorDetails } from './getErrorDetails';
+import { getValueAndTypeFromMessage } from './getValueAndTypeFromMessage';
+import { buildStackFrame } from './stackFrames';
 
 export class ErrorsInstrumentation extends BaseInstrumentation {
```

Comment:
This does not look properly indented. Missing semicolons as well

Reply:
hmm, there seems to be a general problem with linter / CI, as build is green despite total anarchy with semicolons :D 
will check

- [ ] check linter/ci